go-controller/pkg/ovn/master_test.go

```diff
@@ -1678,6 +1679,103 @@ var _ = ginkgo.Describe("Default network controller operations", func() {
 		})
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
 	})
+
+	ginkgo.It("verify IPs in host-network-namespace address_set after remote zone node add or delete", func() {
+		app.Action = func(ctx *cli.Context) error {
+			_, err := config.InitConfig(ctx, nil, nil)
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+			config.OVNKubernetesFeature.EnableInterconnect = true
+			config.Kubernetes.HostNetworkNamespace = "ovn-host-network"
+
+			// PortBinding is required to add interconect resources for
+			// remote zone nodes
+			pb := &sbdb.PortBinding{
+				LogicalPort: "tstor-node1",
+				Datapath:    "tstor-node1",
+				TunnelKey:   5,
+			}
+			dbSetup.SBData = append(
+				dbSetup.SBData,
+				pb,
+			)
+			// transit_switch is required to be able to start OVN with an
+			// empty NodeList
```

Comment:
add some more info here so that we can remember why so... "to avoid calling ensure transit switch which expects a list of nodes" etc.. let's say the code base changes tomorrow this comment will become irrelevant..

Reply:
Taken care